src/tests/topp/FileInfo_19_output.txt

```diff
@@ -7,13 +7,13 @@ File type: mzML
 Instrument: 
 
 MS levels: 1, 2
-Total number of peaks: 2679
+Total number of peaks: 2681
 Number of spectra: 2
```

Comment:
What changed here?

Reply:
I need to check. Most probably the TIC as it would add one peak per spectrum.